addons/ai/fnc_taskDefend.sqf

```diff
@@ -58,7 +58,7 @@ private _statics = _position nearObjects ["StaticWeapon", _radius];
 private _buildings = _position nearObjects ["Building", _radius];
 
 // Filter out occupied statics
-_statics = _statics select {(_x emptyPositions "Gunner") > 0};
+_statics = _statics select {locked _x != 2 && {(_x emptyPositions "Gunner") > 0}};
```

Comment:
0 - Unlocked; 1 - Default; 2 - Locked; 3 - Locked for player; -1 - vehicle is null.

Reply:
You want to also check for -1?

Reply:
No, but this is really messy.